src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -804,9 +806,16 @@
                   :in csr-stream
                   :env (merge env {"RUBYLIB" rubylib})})]
     (log/debugf "Autosign command '%s %s' exit status: %d"
-                executable subject (:exit results))
+                executable subject (:exit-code results))
     (log/debugf "Autosign command '%s %s' output: %s"
-                executable subject (str (:err results) (:out results)))
+                executable subject (str (:stderr results) (:stdout results)))
```

Comment:
I could consolidate the two and log at warn if stderr is generated and debug if no stderr is generated, but the value is that if there is stderr, the user won't need to adjust their loglevel to see the log message.